rotatingtext/src/main/java/com/sdsmdg/harjot/rotatingtext/RotatingTextWrapper.java

```diff
@@ -287,8 +287,8 @@ public void addWord(int rotatableIndex, int wordIndex, String newWord) {
         }
     }
 
-    
-  
+
+
```

Comment:
Undo

Reply:
I have not made any changes in this class but to ensure this I have copied and pasted code from the original code I had cloned.

Reply:
@CoderMayhem Can you help get this resolved in the and the other PR.

Reply:
@Anu-123-gif try to physically undo these changes by deleting the lines and then make a commit to see if it works. This file should not appear in the "files changed" section.

Reply:
@dev-ritik I am not sure about this. I guess copying and pasting from the main code as done by @Anu-123-gif should have fixed this. Also, @Anu-123-gif make a new commit to undo the redundant change in the PR#28.

Reply:
I have removed the redundant changes which I could....